src/GitVersionExe/ExtensionMethods.cs

```diff
@@ -7,8 +7,8 @@ namespace GitVersion
 {
     public static class ExtensionMethods
```

Comment:
I think we just use `Extensions` as the class name for extension methods elsewhere in the codebase, perhaps with a clarifying prefix. Since this seems to deal with `string`, perhaps `StringExtensions` is a better name?

Reply:
Moved `ExtensionMethods` to `GitVersionCore` and renamed to `StringExtensions`